lib/ash/actions/read/read.ex

```diff
@@ -178,6 +178,8 @@ defmodule Ash.Actions.Read do
 
     query = load_and_select_sort(query)
 
+    query = add_relationship_count_aggregates(query)
```

Comment:
If this is now done here shouldn't it be removed from line 435?

Reply:
yep :)

Reply:
I'll do that and push it up